pkg/utils/kubeclient/pod.go

```diff
@@ -46,7 +46,7 @@ func IsFailedPod(pod *corev1.Pod) bool {
 	return pod != nil && pod.Status.Phase == corev1.PodFailed
 }
 
-// get pod given name and namespace of the pod.
+// GetPodByName gets pod given name and namespace of the pod.
```

Comment:
GetPodByName gets pod with given name and namespace of the pod.

Reply:
Fixed.